### ISSUE TITLE
Schema doesnt match response

### ISSUE THREAD

**Issue by Netizine** [openai/openai-openapi#7]
Take the following edit with the following post values for CreateEditRequest
{
	"model": "text-davinci-edit-001",
	"input": "What day of the wek is it?",
	"instruction": "Fix the spelling mistakes.",
	"n": 1,
	"temperature": 1,
	"top_p": 1
}
The response I get for CreateEditResponse is as follows:
{
  "object": "edit",
  "created": 1671267871,
  "choices": [
    {
      "text": "What day of the week is it?\n",
      "index": 0
    }
  ],
  "usage": {
    "prompt_tokens": 26,
    "completion_tokens": 28,
    "total_tokens": 54
  }
}

The problem is that the response contains no id, yet in the spec for CreateEditResponse, id is defined as a required field.

**Comment by janodev** [comment]
`model` too I’m afraid.

Response to the sample edit at https://beta.openai.com/docs/api-reference/edits/create
outputs the following JSON
```json
{
    "object": "edit",
    "created": 1672191563,
    "choices": [
        {
            "text": "What day of the week is it?\n",
            "index": 0
        }
    ],
    "usage": {
        "prompt_tokens": 25,
        "completion_tokens": 28,
        "total_tokens": 53
    }
}
```
which misses `id` and `model` of the [CreateEditResponse](https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L2137). Easily fixed by removing those properties from the required list at lines [2193](https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L2193) and [2196](https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L2196).

**Comment by om4james** [comment]
> which misses `id` and `model` of the [CreateEditResponse](https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L2137). Easily fixed by removing those properties from the required list at lines [2193](https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L2193) and [2196](https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L2196).

I have implemented that change in https://github.com/openai/openai-openapi/pull/16. That PR removes the two properties from the schema definition and the `required` list.

**Comment by Netizine** [comment]
I agree that they easy to remove but if you look at all the other responses, it creates inconsistencies in the expected replies. It makes the pattern easier to model if they were return as the response is more consistent then.

**Comment by schnerd** [comment]
Thanks folks! Agree that having a more consistent response format would be better here – I'll work with the team on that.

In the meantime, we'll remove those properties from the spec by merging @om4james's PR.
